src/lib/i18n/en_US.ts

```diff
@@ -1,10 +1,25 @@
 export default {
-  "weather-data": {
-    label: {
-      name: "Weather"
+    "weather-data": {
+        label: {
+            name: "Weather",
+        },
+        title: {
+            name: "German Weather Service",
```

Comment:
translation may not be correct
```suggestion
            name: "German Meteorological Service",
```